### PR TITLE
Fix match theme by handle with cannonical prefix https://hdl.handle.net/ not working

### LINKED ISSUE #2348: Bug: theme matching by handle doesn't work, but uuid does
**Describe the bug**
Matching on a collection handle does not appear to work for a collection, but using a UUID does work (except for the duplicated elements #2346)

**To Reproduce**

```yml
themes:
  - name: etdr
    extends: krex
    handle: '2097/4'
    # uuid: '6d684812-6d29-4564-8b51-f5670957936b'

  # default theme, based on DSpace
  - name: krex
```

This works though
```yml
themes:
  - name: etdr
    extends: krex
    uuid: '6d684812-6d29-4564-8b51-f5670957936b'

  # default theme, based on DSpace
  - name: krex
```

**Expected behavior**
Theming should be applied to the collection items when using the handle, or the uuid.

**Related work**

### PR DESCRIPTION
**Pull request by alexandrevryghem**
## References
* Fixes DSpace/dspace-angular#2348
* Related to DSpace/dspace-angular#2562

## Description
This PR exposes the `handle.canonical.prefix` to the frontend, in order to use it to parse handles.

## Instructions for Reviewers
List of changes in this PR:
* Exposed the `handle.canonical.prefix` in `dspace.cfg`
* Removed the line breaks in `default.license` in order to make angular decide when lines should break. This way the license section is more responsive in the submission forms. This problem is a side effect of DSpace/dspace-angular#2229

## Checklist
- [x] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & integration tests). Exceptions may be made if previously agreed upon.
- [x] My PR passes Checkstyle validation based on the [Code Style Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Style+Guide).
- [x] My PR includes Javadoc for _all new (or modified) public methods and classes_. It also includes Javadoc for large or complex private methods.
- [x] My PR passes all tests and includes new/updated Unit or Integration Tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [x] If my PR includes new libraries/dependencies (in any `pom.xml`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.
- [x] If my PR modifies REST API endpoints, I've opened a separate [REST Contract](https://github.com/DSpace/RestContract/blob/main/README.md) PR related to this change.
- [x] If my PR includes new configurations, I've provided basic technical documentation in the PR itself.
- [x] If my PR fixes an issue ticket, I've [linked them together](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

### CODE REVIEW

File: dspace/config/default.license [hunk 1]
```diff
@@ -3,34 +3,16 @@ This sample license is provided for informational purposes only.
...
-
-IF THE SUBMISSION IS BASED UPON WORK THAT HAS BEEN SPONSORED OR SUPPORTED
-BY AN AGENCY OR ORGANIZATION OTHER THAN DSU, YOU REPRESENT THAT YOU HAVE
-FULFILLED ANY RIGHT OF REVIEW OR OTHER OBLIGATIONS REQUIRED BY SUCH
-CONTRACT OR AGREEMENT.
-
-DSU will clearly identify your name(s) as the author(s) or owner(s) of the
-submission, and will not make any alteration, other than as allowed by this
-license, to your submission.
+By signing and submitting this license, you (the author(s) or copyright owner) grants to DSpace University (DSU) the non-exclusive right to reproduce, translate (as defined below), and/or distribute your submission (including the abstract) worldwide in print and electronic format and in any medium, including but not limited to audio or video.
```
Comment (tdonohue) [inline dspace/config/default.license:6]: @alexandrevryghem : Why did you modify the `default.license` in this PR?  It seems to be an unrelated change?  I suspect this was accidental and needs to be reverted.

Comment (alexandrevryghem) [reply dspace/config/default.license:6]: I did it on purpose, but I can revert it 😅. In PR [#2229](https://github.com/DSpace/dspace-angular/pull/2229) the way the `default.license` was rendered has been changed to also display new lines. The problem is that because we added manual new line breaks in that file, those line breaks are also being displayed. This results in the lines not wrapping correctly on some screens. I can move this to a different PR if you prefer, but I thought that the change was small enough to add it to this PR 🤷‍♂️
image
image

**Review by tdonohue — APPROVED** [review]
👍 Thanks @alexandrevryghem !  Looks good & I tested this with the frontend PR.  Also verified the changes to the linebreaks in `default.license` work well.  This will be ready to merge as soon as the frontend PR can be fixed up (see my [review there](https://github.com/DSpace/dspace-angular/pull/2562#pullrequestreview-1723570507)).
